src/ui/oled/ssd1312.c

```diff
@@ -118,6 +120,8 @@ void ssd1312_update(void)
     /* The SSD1312 has one page per 8 rows. One page is 128 bytes. Every byte is 8 rows */
     for (size_t i = 0; i < 64 / 8; i++) {
         oled_writer_write_cmd(SSD1312_CMD_SET_PAGE_START_ADDRESS(i));
```

Comment:
Could you add a comment that this only has been experienced on a single NOVA device? Because this fix fixes the symptom and not the underlying issue.

The column address is autoincremented for every access so if we end up with an off-by-one or more it means not all bytes arrived at the screen.

Reply:
Done, sounds good? Not sure if saying it was one Nova helps, it could probably be reproduced on other novas (or even bb02s), but reproducing is a pain so we haven't tried on more devices.

Any idea for the actual cause?

Reply:
could be faulty hardware